### PR TITLE
[All Hosts] (browsers) restore info about browser switch command and …

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
…other small fixes

Fixes [3023](https://github.com/OfficeDev/office-js-docs-pr/issues/3023)

### CODE REVIEW

File: docs/includes/use-legacy-edge-or-ie.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+If your project is nodejs-based; that is, not developed with Visual Studio and Internet Informaton server (IIS); you can force Office on Windows to use Edge Legacy or Internet Explorer to run add-ins, even if you have a combination of Windows and Office versions that would normally use a more recent browser. For more information about which browsers are used by various combinations of Windows and Office versions, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/includes/use-legacy-edge-or-ie.md:1]:
```suggestion
If your project is node.js-based (that is, not developed with Visual Studio and Internet Information server (IIS)), you can force Office on Windows to use Edge Legacy or Internet Explorer to run add-ins, even if you have a combination of Windows and Office versions that would normally use a more recent browser. For more information about which browsers are used by various combinations of Windows and Office versions, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

File: docs/includes/use-legacy-edge-or-ie.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    npm install office-addin-dev-settings --save-dev
+    ```
+
+1. Specify the browser that you want Office to use with the following command in a command prompt in the root of the project. Replace `<path-to-manifest>` with the relative path, which is just the manifest filename if it is in the root of the project. Replace `<webview>` with either `ie` or `edge-legacy`.
+
+    ```command&nbsp;line
+    npx office-addin-dev-settings webview <path-to-manifest> <webview>
+    ```
+
+    The following is an example:
```
Comment (ElizabethSamuel-MSFT) [inline docs/includes/use-legacy-edge-or-ie.md:15]:
```suggestion
    The following is an example.
```

File: docs/includes/use-legacy-edge-or-ie.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+    ```command&nbsp;line
+    npx office-addin-dev-settings webview manifest.xml ie
+    ```
+
+    You should see a message in the command line that the webview type is now set to IE (or Edge Legacy).
+
+1. When you're finished, set Office to resume using the default browser for your combination of Windows and Office versions with the following command:
```
Comment (ElizabethSamuel-MSFT) [inline docs/includes/use-legacy-edge-or-ie.md:23]:
```suggestion
1. When you're finished, set Office to resume using the default browser for your combination of Windows and Office versions with the following command.
```

File: docs/testing/debug-add-ins-using-devtools-edge-legacy.md [hunk 4]
```diff
@@ -57,12 +57,24 @@ To determine which browser you're using, see [Browsers used by Office Add-ins](.
...
 1. Select the process to open it and debug just as described in the section [Debug a task pane add-in using Microsoft Edge DevTools Preview](#debug-a-task-pane-add-in-using-microsoft-edge-devtools-preview).
 
    :::image type="content" source="../images/edge-devtools-with-add-in-and-dialog-processes.png" alt-text="Screenshot of Edge DevTools showing a process named My Dialog.":::
 
-## Install a version of Office that uses Microsoft Edge Legacy
+## Switch to the Edge Legacy webview
 
-[!INCLUDE [Steps to install Office that uses Edge Legacy or Internet Explorer](../includes/install-office-that-uses-legacy-edge-or-ie.md)]
\ No newline at end of file
+There are two ways to switch the Edge Legacy webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Edge Legacy by default. We recommend the first method. But you should use the second in the following scenarios:
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-devtools-edge-legacy.md:68]:
```suggestion
There are two ways to switch the Edge Legacy webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Edge Legacy by default. We recommend the first method. But you should use the second in the following scenarios.
```

File: docs/testing/debug-add-ins-using-devtools-edge-legacy.md [hunk 5]
```diff
@@ -57,12 +57,24 @@ To determine which browser you're using, see [Browsers used by Office Add-ins](.
...
+
+- Your project was developed with Visual Studio and IIS. It isn't nodejs-based.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-devtools-edge-legacy.md:70]:
```suggestion
- Your project was developed with Visual Studio and IIS. It isn't node.js-based.
```

File: docs/testing/debug-add-ins-using-devtools-edge-legacy.md [hunk 6]
```diff
@@ -57,12 +57,24 @@ To determine which browser you're using, see [Browsers used by Office Add-ins](.
...
+- You want to be absolutely robust in your testing.
+- If for any reason the command line tool doesn't work.
+
+### Switch on the command line
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-devtools-edge-legacy.md:74]: I'm not sure about saying "on" - it sounds like turning on the command line which I don't think is the intention. Maybe "via" or "using"...

Here and other instances

```suggestion
### Switch via the command line
```

File: docs/testing/debug-add-ins-using-f12-tools-ie.md [hunk 7]
```diff
@@ -79,7 +79,19 @@ If your add-in uses the Office Dialog API, the dialog runs in a separate process
 1. Open the dialog and then select the **Refresh** button in the tools. The dialog process is shown. Its name is the file name of the file that is open in the dialog.
 1. Select the process to open it and debug just as described in the section [Debug a task pane add-in using the F12 tools](#debug-a-task-pane-add-in-using-the-f12-tools).
 
-## Install a version of Office that uses Internet Explorer
+## Switch to the Internet Explorer 11 webview
+
+There are two ways to switch the Internet Explorer webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Internet Explorer by default. We recommend the first method. But you should use the second in the following scenarios:
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-f12-tools-ie.md:84]:
```suggestion
There are two ways to switch the Internet Explorer webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Internet Explorer by default. We recommend the first method. But you should use the second in the following scenarios.
```

File: docs/testing/debug-add-ins-using-f12-tools-ie.md [hunk 8]
```diff
@@ -79,7 +79,19 @@ If your add-in uses the Office Dialog API, the dialog runs in a separate process
...
+
+- Your project was developed with Visual Studio and IIS. It isn't nodejs-based.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-f12-tools-ie.md:86]:
```suggestion
- Your project was developed with Visual Studio and IIS. It isn't node.js-based.
```

File: docs/testing/ie-11-testing.md [hunk 9]
```diff
@@ -30,40 +30,25 @@ If you plan to market your add-in through AppSource or you plan to support older
...
+> [!TIP]
+> [!INCLUDE[Identify the webview through the add-in UI](../includes/identify-webview-in-ui.md)]
 
-1. In the root folder of your project, run the following in the command line. This example assumes that your project's manifest file is in the root. If it isn't, specify the relative path to the manifest file. You should see a message in the command line that the web view type is now set to IE.
+There are two ways to switch the Internet Explorer webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Internet Explorer by default. We recommend the first method. But you should use the second in the following scenarios:
 
-    ```command&nbsp;line
-    npx office-addin-dev-settings webview manifest.xml ie
-    ```
+- Your project was developed with Visual Studio and IIS. It isn't nodejs-based.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/ie-11-testing.md:40]:
```suggestion
- Your project was developed with Visual Studio and IIS. It isn't node.js-based.
```

File: docs/testing/ie-11-testing.md [hunk 10]
```diff
@@ -30,40 +30,25 @@ If you plan to market your add-in through AppSource or you plan to support older
...
-<!-- The npx command is broken. Save this text for reuse if it is ever fixed.
-1. Create a Yo Office generator project. It doesn't matter what kind of project you select, this tooling will work with all project types.
-
-    > [!NOTE]
-    > If you have an existing project and want to add this tooling without creating a new project, skip this step and move to the next step. 
+> [!TIP]
+> [!INCLUDE[Identify the webview through the add-in UI](../includes/identify-webview-in-ui.md)]
 
-1. In the root folder of your project, run the following in the command line. This example assumes that your project's manifest file is in the root. If it isn't, specify the relative path to the manifest file. You should see a message in the command line that the web view type is now set to IE.
+There are two ways to switch the Internet Explorer webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Internet Explorer by default. We recommend the first method. But you should use the second in the following scenarios:
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/ie-11-testing.md:38]:
```suggestion
There are two ways to switch the Internet Explorer webview. You can run a simple command in a command prompt, or you can install a version of Office that uses Internet Explorer by default. We recommend the first method. But you should use the second in the following scenarios.
```

File: docs/testing/debug-add-ins-using-devtools-edge-legacy.md [hunk 11]
```diff
@@ -57,12 +57,24 @@ To determine which browser you're using, see [Browsers used by Office Add-ins](.
...
 
-1. Run the add-in and the tools. 
+1. Run the add-in and the tools.
 1. Open the dialog and then select the **Refresh** button in the tools. The dialog process is shown. Its name comes from the `<title>` element in the HTML file that is open in the dialog.
 1. Select the process to open it and debug just as described in the section [Debug a task pane add-in using Microsoft Edge DevTools Preview](#debug-a-task-pane-add-in-using-microsoft-edge-devtools-preview).
 
    :::image type="content" source="../images/edge-devtools-with-add-in-and-dialog-processes.png" alt-text="Screenshot of Edge DevTools showing a process named My Dialog.":::
 
-## Install a version of Office that uses Microsoft Edge Legacy
+## Switch to the Edge Legacy webview
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-devtools-edge-legacy.md:67]: The content in this section seems to be the same (or pretty similar) to that in the other files. Any reason not to use an include?

File: docs/testing/debug-add-ins-using-f12-tools-ie.md [hunk 12]
```diff
@@ -79,7 +79,19 @@ If your add-in uses the Office Dialog API, the dialog runs in a separate process
...
+- You want to be absolutely robust in your testing.
+- If for any reason the command line tool doesn't work.
+
+### Switch on the command line
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-f12-tools-ie.md:90]: see earlier comment about "on"

File: docs/testing/ie-11-testing.md [hunk 13]
```diff
@@ -30,40 +30,25 @@ If you plan to market your add-in through AppSource or you plan to support older
...
-    ```command&nbsp;line
-    npx office-addin-dev-settings webview manifest.xml ie
-    ```
+- Your project was developed with Visual Studio and IIS. It isn't nodejs-based.
+- You want to be absolutely robust in your testing.
+- If for any reason the command line tool doesn't work.
 
-> [!TIP]
-> It isn't necessary to use this command, but it should help debug the majority of issues related to the Internet Explorer 11 runtime. For complete robustness, you should test using computers with various combinations of Windows 7, 8.1, 10, and 11 and various versions of Office. For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md) and [How to revert to an earlier version of Office](https://support.microsoft.com/topic/2bd5c457-a917-d57e-35a1-f709e3dda841).
+### Switch on the command line
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/ie-11-testing.md:44]: see earlier comment about "on"

File: docs/testing/debug-add-ins-using-devtools-edge-legacy.md (hunk 11 above)
Comment (Rick-Kirkham) [reply docs/testing/debug-add-ins-using-devtools-edge-legacy.md:67]: It is using includes. But the headings and opening paragraph need to be different. One article is about Edge Legacy and others are about IE.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**